lib/user.rb

```diff
@@ -0,0 +1,58 @@
+class User 
+  attr_reader :user_id, :name, :username, :email, :password
```

Comment:
also - i think a lot of web frameworks either have something built in or have a popular compatible library which deals with [password hashing ](https://learn.co/lessons/sinatra-password-security) so that you don't actually have to store the password at all, and instead just a hash of it